tasks/src/main/scala/dagr/tasks/gatk/Mutect1.scala

```diff
@@ -51,6 +51,8 @@ class Mutect1(val tumorBam: PathToBam,
              ) extends GatkTask(walker="MuTect", ref=ref, intervals=Some(intervals)) {
 
   override protected def addWalkerArgs(buffer: ListBuffer[Any]): Unit = {
+    require(gatkMajorVersion < 4, s"Mutect v1 is not supported in GATK v${gatkMajorVersion}.")
```

Comment:
This `require` is in the `addWalkerArgs` method, but in `IndelRealigner` and `RealignerTargetCreator` it's in the constructor.  The former will check when we run the task, while the latter when we build the task.  Up to you.

Reply:
Good call.  I'll move this one up because failing earlier seems better.